src/literal.h

```diff
@@ -105,6 +105,11 @@ class Literal {
   Literal convertSToF64() const;
   Literal convertUToF64() const;
 
+  Literal truncSatToSI32() const;
```

Comment:
maybe add a TODO here that we should rename the above `convert*ToF*` methods to `trunc*ToF*`, since these new methods are their saturating parallels so they should have similar names (but those older methods appear to be poorly named, or rather named the old wasm way).

Reply:
I went ahead and renamed them since it was a relatively small change.